ui/src/app.js

```diff
@@ -123,6 +125,7 @@ function processData (data) {
   if (data.tx && data.tx.discount) setCurrentDiscount(data.tx.discount)
   if (data.receiptStatus) setReceiptPrint(data.receiptStatus, null)
   if (data.smsReceiptStatus) setReceiptPrint(null, data.smsReceiptStatus)
+  if (data.automaticPrint) setAutomaticPrint(data.automaticPrint)
```

Comment:
Shouldn't `setAutomaticPrint()` be run unconditionally here? There's an if inside on `data.automaticPrint` already.

Reply:
That would make the `else` part be executed everytime, which we do not want

Reply:
Hmm I don't really understand it, but you say it's relevant, so be it.